src/repos/dfusion/DfusionRepoImpl.ts

```diff
@@ -26,14 +26,14 @@ export class DfusionRepoImpl implements DfusionRepo {
   public watchOrderPlacement (params: WatchOrderPlacementParams) {
     this._contract.events
       .OrderPlacement({}, error => params.onError(error))
-      .on('connected', () => {
-        log.info('Starting to listen for new orders')
+      .on('connected', subscriptionId => {
+        log.debug('Starting to listen for new orders. SubscriptionId: %s', subscriptionId)
```

Comment:
question: any special reason for `%s` rather than string templates?

Reply:
Yes!, no a super big one though. The log is not necessarily enabled, so if you interpolate and then call the fn, then you interpolate for nothing :) 

Anyways, as I said, I don't care that much about optimizing at that level, so sometimes I use string interpolation if I feel that the code is more readable, what I find more important.

Reply:
Got it!